diff -ruN wxsvg-1.5.20.orig/svgview/Makefile.in wxsvg-1.5.20/svgview/Makefile.in
--- wxsvg-1.5.20.orig/svgview/Makefile.in	2019-08-30 12:32:50.634006900 -0700
+++ wxsvg-1.5.20/svgview/Makefile.in	2019-08-30 12:37:38.444831000 -0700
@@ -411,7 +411,7 @@
 
 svgview$(EXEEXT): $(svgview_OBJECTS) $(svgview_DEPENDENCIES) $(EXTRA_svgview_DEPENDENCIES) 
 	@rm -f svgview$(EXEEXT)
-	$(AM_V_CXXLD)$(CXXLINK) $(svgview_OBJECTS) $(svgview_LDADD) $(LIBS)
+	$(AM_V_CXXLD)$(CXXLINK) $(svgview_OBJECTS) $(svgview_LDADD) $(LIBS) -l:libharfbuzz.a
 
 mostlyclean-compile:
 	-rm -f *.$(OBJEXT)
